### DISCUSSION TITLE
Question about section of /tutorial/handling-errors on RequestValidationError

### DISCUSSION THREAD

**Discussion by oskarkv** [fastapi/fastapi#5767] [Questions]
Beginning here https://fastapi.tiangolo.com/tutorial/handling-errors/#override-request-validation-exceptions the tutorial says:

> When a request contains invalid data, FastAPI internally raises a RequestValidationError.

And a bit further down:

> RequestValidationError vs ValidationError
>
> RequestValidationError is a sub-class of Pydantic's ValidationError.
>
> FastAPI uses it so that, if you use a Pydantic model in response_model, and your data has an error, you will see the error in your log.
>
> But the client/user will not see it. Instead, the client will receive an "Internal Server Error" with a HTTP status code 500.

Now I'm confused. First of all, there seems to be some sort of mistake here. The error is called `RequestValidationError` but apparently it has to do with `response_model`. And it says that I will see an error in my log. Well, how does that work? What log and what exception handler is used to make it happen?

**Comment by YuriiMotov** [comment]
Docs has been updated since then and there is no that confusing statement anymore
